tuning/trackers/tracker_factory.py

```diff
@@ -81,6 +104,8 @@ def _register_trackers():
         _register_aim_tracker()
     if FILE_LOGGING_TRACKER not in REGISTERED_TRACKERS:
         _register_file_logging_tracker()
+    if MLFLOW_TRACKER not in REGISTERED_TRACKERS:
```

Comment:
For completeness, could you also add a comment below line 100?

Reply:
sure!